src/js/letters/containers/AddressSection.jsx

```diff
@@ -4,15 +4,17 @@ import Scroll from 'react-scroll';
 
 import { scrollToFirstError } from '../../common/utils/helpers';
 import LoadingIndicator from '../../common/components/LoadingIndicator';
+import Modal from '../../common/components/Modal';
 
 import {
+  addressModalContent,
+  addressUpdateUnavailable,
   getStateName,
   getZipCode,
+  inferAddressType,
   isDomesticAddress,
-  isMilitaryAddress,
   isInternationalAddress,
-  addressUpdateUnavailable,
-  inferAddressType,
```

Comment:
Took me way too long to figure out this just meant they were being rearranged. I was starting to think maybe there was some black magic goings on all up in these parts. 😨

Reply:
Yeah sorry, when the list starts to grow out of control I like to alphabetize things. I meant to put a comment on this line to clarify; and then...I didn't.